lib/task-data/tasks/install-coreos.js

```diff
@@ -17,7 +17,17 @@ module.exports = {
         installDisk: '/dev/sda',
         repo: '{{file.server}}/coreos',
         version: 'current',
-        rootPassword: "RackHDRocks!"
+        rootPassword: "RackHDRocks!",
+
+        progressMilestones: {
+            //jshint ignore: start
+            requestProfile:     { value: 1, description: 'Enter ipxe and request OS installation profile' },
+            enterProfile:       { value: 2, description: 'Enter profile, start to download installer'},
+            bootInstaller:      { value: 3, description: 'Boot into installer' },
+            executeInstallation:{ value: 4, description: 'Execute OS installation'},
```

Comment:
This milestone is put before following command (see install-coreos.sh):
```
sudo coreos-install -d <%=installDisk%> -c $CLOUD_CONFIG_FILE -b <%=repo%>
```
From the command name itself `coreos-install`, This is not for configuration, this is to execute the coreos installation. This is the reason why I choose this word.